tox.ini

```diff
@@ -28,6 +28,8 @@ deps =
 commands =
     black --check --diff .
     isort --skip-gitignore --check-only --diff src tests
+    # typing stubs for orjson exist but I don't want mypy to use them!
+    pip uninstall -y orjson
```

Comment:
apparently `import orjson  # type: ignore` is not enough, mypy will still try to use the available orjson stubs to typecheck. I don't want to use them, because I want to support both (or rather a subset of) orjson and built-in json and these have slightly different api. Since I shadow `orjson as json` I would have to `type: ignore` every single line.

Reply:
Maybe using `cast` would help?

Reply:
cast to what?

Reply:
done [aa40347](https://github.com/fonttools/ufoLib2/pull/230/commits/aa403479622e93e809dd089ac99981c6b7989fe3)